Erro ao rodar make test-unit apontando para o sei4

Clone o projeto.
Siga o manual para configurar o ambiente para testes funcionais.
Execute
``` 
make up
make install
make test-unit
``` 

Vai aparecer o seguinte erro:

```
+ make test-unit

docker-compose -f tests_sei4/funcional/docker-compose.yml run --rm -w /tests php-test-unit bash -c 'vendor/bin/phpunit'

Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(

PHPUnit 5.7.27 by Sebastian Bergmann and contributors.



Usage: phpunit [options] UnitTest [UnitTest.php]

       phpunit [options] <directory>



Code Coverage Options:



  --coverage-clover <file>  Generate code coverage report in Clover XML format.

  --coverage-crap4j <file>  Generate code coverage report in Crap4J XML format.

  --coverage-html <dir>     Generate code coverage report in HTML format.

  --coverage-php <file>     Export PHP_CodeCoverage object to file.

  --coverage-text=<file>    Generate code coverage report in text format.

                            Default: Standard output.

  --coverage-xml <dir>      Generate code coverage report in PHPUnit XML format.

  --whitelist <dir>         Whitelist <dir> for code coverage analysis.

  --disable-coverage-ignore Disable annotations for ignoring code coverage.



Logging Options:



  --log-junit <file>        Log test execution in JUnit XML format to file.

  --log-teamcity <file>     Log test execution in TeamCity format to file.

  --testdox-html <file>     Write agile documentation in HTML format to file.

  --testdox-text <file>     Write agile documentation in Text format to file.

  --testdox-xml <file>      Write agile documentation in XML format to file.

  --reverse-list            Print defects in reverse order



Test Selection Options:



  --filter <pattern>        Filter which tests to run.

  --testsuite <name>        Filter which testsuite to run.

  --group ...               Only runs tests from the specified group(s).

  --exclude-group ...       Exclude tests from the specified group(s).

  --list-groups             List available test groups.

  --list-suites             List available test suites.

  --test-suffix ...         Only search for test in files with specified

                            suffix(es). Default: Test.php,.phpt



Test Execution Options:



  --report-useless-tests    Be strict about tests that do not test anything.

  --strict-coverage         Be strict about @covers annotation usage.

  --strict-global-state     Be strict about changes to global state

  --disallow-test-output    Be strict about output during tests.

  --disallow-resource-usage Be strict about resource usage during small tests.

  --enforce-time-limit      Enforce time limit based on test size.

  --disallow-todo-tests     Disallow @todo-annotated tests.



  --process-isolation       Run each test in a separate PHP process.

  --no-globals-backup       Do not backup and restore $GLOBALS for each test.

  --static-backup           Backup and restore static attributes for each test.



  --colors=<flag>           Use colors in output ("never", "auto" or "always").

  --columns <n>             Number of columns to use for progress output.

  --columns max             Use maximum number of columns for progress output.

  --stderr                  Write to STDERR instead of STDOUT.

  --stop-on-error           Stop execution upon first error.

  --stop-on-failure         Stop execution upon first error or failure.

  --stop-on-warning         Stop execution upon first warning.

  --stop-on-risky           Stop execution upon first risky test.

  --stop-on-skipped         Stop execution upon first skipped test.

  --stop-on-incomplete      Stop execution upon first incomplete test.

  --fail-on-warning         Treat tests with warnings as failures.

  --fail-on-risky           Treat risky tests as failures.

  -v|--verbose              Output more verbose information.

  --debug                   Display debugging information during test execution.



  --loader <loader>         TestSuiteLoader implementation to use.

  --repeat <times>          Runs the test(s) repeatedly.

  --teamcity                Report test execution progress in TeamCity format.

  --testdox                 Report test execution progress in TestDox format.

  --testdox-group           Only include tests from the specified group(s).

  --testdox-exclude-group   Exclude tests from the specified group(s).

  --printer <printer>       TestListener implementation to use.



Configuration Options:



  --bootstrap <file>        A "bootstrap" PHP file that is run before the tests.

  -c|--configuration <file> Read configuration from XML file.

  --no-configuration        Ignore default configuration file (phpunit.xml).

  --no-coverage             Ignore code coverage configuration.

  --no-extensions           Do not load PHPUnit extensions.

  --include-path <path(s)>  Prepend PHP's include_path with given path(s).

  -d key[=value]            Sets a php.ini value.

  --generate-configuration  Generate configuration file with suggested settings.



Miscellaneous Options:



  -h|--help                 Prints this usage information.

  --version                 Prints the version and exits.

  --atleast-version <min>   Checks that version is greater than min and exits.

make: *** [test-unit] Error 2

script returned exit code 2
```

--- a/Makefile
+++ b/Makefile
@@ -180,7 +180,7 @@ test-parallel-otimizado:
 	
 	
 test-unit: $(FILE_VENDOR_UNITARIO)
-	$(CMD_DOCKER_COMPOSE) -f $(PEN_TEST_FUNC)/docker-compose.yml run --rm -w /tests php-test-unit bash -c 'vendor/bin/phpunit'
+	$(CMD_DOCKER_COMPOSE) -f $(PEN_TEST_FUNC)/docker-compose.yml run --rm -w /tests php-test-unit bash -c 'vendor/bin/phpunit rn/ProcessoEletronicoRNTest.php'
 
 test: test-unit test-functional
 

